scripts/run-sync.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+const uploadAPISpecification = require('./upload-api-specification');
+const core = require('@actions/core');
+const github = require('@actions/github');
```

Comment:
Do we need this?

Reply:
we need core to make sure the GH action fails when there is some error in our logic. I'll remove the github import..